content/docs/what-is-kudo.md

```diff
@@ -133,3 +133,38 @@ Normally, _Operator_ and _OperatorVersion_ are handled by the manager and rarely
 ```bash
 $ kubectl kudo install kafka --instance dev-kafka
 ```
+
+## Executing Plans
+
+In general, KUDO manager will automatically [execute a plan](what-is-kudo.md#operator-parameters) when the corresponding parameter changes. However, sometimes this is not enough. Sometimes you need to execute a plan manually like running a periodic `backup` plan or executing `restore` plan in case of data corruption. Such plans typically do not need a corresponding parameter. KUDO [v0.11.0](https://github.com/kudobuilder/kudo/releases/tag/v0.11.0) introduced new feature: manual plan execution. In a nutshell, you can now:
+
+```bash
+$ kubectl kudo plan trigger --name deploy --instance my-instance
+```
+
+which will execute `deploy` plan on `my-instance`. While this looks relatively easy on the surface, the devil is in detail, so let's take a closer look.
+
+### Plan Life Cycle
+
+Having the ability to trigger multiple plans on demand raises the question: what happens if two plans run concurrently? The answer is: it depends. Should two plans be completely independent of each other (e.g. `deploy` deploys the services and `monitor` deploys the monitoring pods) both can be executed in parallel. But if two plans in question are `backup` and `restore` or `deploy` and `migrate`? Some plans are incompatible with others. A few may not even be compatible with themselves. While it is probably ok to restart a running `deploy` plan, a `restore` plan might not be reentrant because of possible data corruption.
```

Comment:
```suggestion
Having the ability to trigger multiple plans on demand raises the question: what happens if two plans run concurrently? The answer is: it depends. Should two plans be completely independent of each other (e.g. `deploy` deploys the services and `monitor` deploys the monitoring pods) both can be executed in parallel. But if two plans in question are `backup` and `restore` or `deploy` and `migrate`? Some plans are incompatible with others. A few may not even be compatible with another running instance of themselves. While it is probably ok to restart a running `deploy` plan, a `restore` plan might not be reentrant because of possible data corruption.
```

Reply:
I reworded it differently.